src/pages/Inventory.page.js

```diff
@@ -10,7 +14,52 @@ export class InventoryPage extends BaseSwagLabPage {
 
     addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
 
+    sortSelect = this.page.getByTestId('product-sort-container');
+
+    inventoryItemPrice = this.page.getByTestId('inventory-item-price');
+
+    inventoryItemTitle = this.page.getByTestId('inventory-item-name');
+
     async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
     }
+
+    async selectSortOption(type) {
+        await this.sortSelect.selectOption({ label: `${type}` });
+    }
+
+    async getProductsNames() {
+        const productNames = (await this.page.locator('[class^="inventory_item_name"]').allInnerTexts()).map((element) => element.trim());
+        return productNames;
+    }
+
+    async getProductPrices() {
+        const productPrices = (await this.page.locator('[class^="inventory_item_price"]').allInnerTexts()).map((element) => parseFloat(element.replace('$', '').trim()));
+        return productPrices;
+    }
+
+    async sortInventoriesTest(sortBy, productNames, productPrices) {
+        switch (sortBy) {
+            case 'Name (A to Z)':
+                const sortedNamesAZ = await this.getProductsNames();
+                const expectedNamesAZ = [...productNames].sort();
```

Comment:
Why do we need the spread operator (...productNames) here?
The `productNames` variable is an array of strings.
The same below